### PR TITLE
'Musical symbols scale' option & more

### PR DESCRIPTION
**Pull request by XiaoMigros**
Resolves: #19366, #17885

This PR changes the following:
- An option for tuplets to use SMuFL number symbols, rather than plain text
- Pedal line text now uses the musical _symbols_ font instead of musical _text_ font.
-  A new text style property, 'Musical symbols scale', to control the size of musical symbols in text-based elements (similar to the existing control for dynamics). This option is available for tuplets, ottava, and pedal lines.
-------
For tuplets, should the using symbols be the default?

### CODE REVIEW

**Comment by oktophonie** [comment]
It should, but it also needs to have a 'Scale' parameter for when people resize them (where the default 100% = 20pt) - like how dynamics now work. When we did dynamics we removed the 'Dynamics' text style (so all settings are now in Style > Dynamics) but we can't do that here because we need to keep the option to use any text font for tuplets. So there's a design question about how to arrange these settings, which are now split between Style > Tuplets and Style > Text styles > Tuplet.

**Comment by XiaoMigros** [comment]
FWIW, the scale is still influenced by the font size.

**Comment by oktophonie** [comment]
Well it shouldn't, and I've just finished writing my [comment](https://github.com/musescore/MuseScore/pull/20774#issuecomment-1872905257) about that on the other PR

File: src/notation/view/widgets/editstyle.ui [hunk 1]
```diff
@@ -6577,6 +6577,13 @@ By default, they will be placed such as that their right end are at the same lev
                     </item>
                    </widget>
                   </item>
+                  <item row="3" column="0" colspan="3">
+                   <widget class="QCheckBox" name="tupletUseSymbols">
+                    <property name="text">
+                     <string>Load number from musical symbols font</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:6583]: Better "Use" rather than "Load"?

**Comment by XiaoMigros** [comment]
Needs rebasing, closing...

**Comment by Jojo-Schmitz** [comment]
You can rebase without closing `git pull --rebase upstream master; git push -f`

**Comment by XiaoMigros** [comment]
It changed my commit order, and as a consequence I could no longer force push the actual changes to the same commit as I had been doing previously.

**Comment by cbjeukendrup** [comment]
It doesn't really matter at all, but just as a hint for the future: force-pushing is something you do with a _branch_, not with a commit. What you probably mean is that you wanted to _amend_ the same commit again and again. Indeed, amending the last commit is easier than amending a non-last commit, but even the latter is possible using an interactive rebase. 
So, you can always force-push a branch, and you can also amend any commit except that amending the last commit is much easier than any other commit. 
However, it seems strange that a rebase changed the order of your commits. A few things to make sure:
- make sure you are _rebasing_ rather than _merging_. When using `git pull`, explicitly specify `--rebase`. Merging creates a merge commit that can indeed get in the way inconveniently.
- make sure you are pulling from `upstream` rather than `origin`, and make sure that `upstream` points to the `musescore/MuseScore` repo and not to the `XiaoMigros/MuseScore` repo.

Anyway, as said, there's not really a problem but I thought you might be happy to know these things for the future.

**Pull request closed without merging**